shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -630,7 +630,8 @@ export default {
                   {{ group.ref.providerDisplay }} &ndash;  {{ group.ref.providerLocation }} / {{ group.ref.providerSize }} ({{ group.ref.providerName }})
                 </div>
               </div>
-              <div v-if="group.ref" class="right mr-45">
+              <div v-if="group.ref" class="right group-header-buttons">
```

Comment:
We should keep the `mr-45` for this element, it looks a bit strange without.

_Before_
![before](https://user-images.githubusercontent.com/40806497/196977231-0a828fab-b96a-4921-a3c2-2acb7b35fb21.png)

_After_
![after](https://user-images.githubusercontent.com/40806497/196977255-1ebdb0ec-ef7f-4482-b605-e8f47e38b994.png)